.eslintrc

```diff
@@ -23,7 +23,13 @@
     "react"
   ],
   "extends": [
-    "eslint:recommended",
+//    "eslint:recommended",
```

Comment:
You should review https://eslint.org/docs/rules/ and see exactly which rules we should apply. `eslint:recommended` is a good starting point though.